src/components/InputComponent/inputComponent.types.d.ts

```diff
@@ -0,0 +1,16 @@
+import { HTMLAttributes } from 'react';
+
+enum Rounded {
+  rounded = 'rounded',
+  normal = 'normal',
+}
+
+/**
+ * Types for Input
+ * @param {string} type - Type of the input
+ */
+
+export interface IInputComponentProps extends HTMLAttributes<HTMLInputElement> {
+  border: Rounded.normal | Rounded.rounded;
+  dark: boolean;
```

Comment:
add `errorMessage` and `error` props here

Reply:
also, rename interface to be IInputProps

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/d1ed5fdefd0af9a627399bf0ab1541da866ae781